lib/segment/src/vector_storage/raw_scorer.rs

```diff
@@ -951,43 +938,13 @@ where
     TVector: ?Sized,
     TQueryScorer: QueryScorer<TVector>,
 {
-    fn score_points(&self, points: &[PointOffsetType], scores: &mut [ScoredPointOffset]) -> usize {
+    fn score_points(&self, scores: &mut [ScoredPointOffset]) {
         if self.is_stopped.load(Ordering::Relaxed) {
-            return 0;
-        }
-        let mut size: usize = 0;
-        for point_id in points.iter().copied() {
-            if !self.check_vector(point_id) {
-                continue;
-            }
-            scores[size] = ScoredPointOffset {
-                idx: point_id,
-                score: self.query_scorer.score_stored(point_id),
-            };
-
-            size += 1;
-            if size == scores.len() {
-                return size;
-            }
-        }
-        size
-    }
-
-    fn score_points_unfiltered(
-        &self,
-        points: &mut dyn Iterator<Item = PointOffsetType>,
-    ) -> Vec<ScoredPointOffset> {
-        if self.is_stopped.load(Ordering::Relaxed) {
-            return vec![];
-        }
-        let mut scores = vec![];
-        for point_id in points {
-            scores.push(ScoredPointOffset {
-                idx: point_id,
-                score: self.query_scorer.score_stored(point_id),
-            });
+            return;
```

Comment:
same here: original semantic is that we return no scored points to prevent further search